css-layout-classes/styles.css

```diff
@@ -0,0 +1,42 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
Your container should only have a `max-width` & `margin` for this one. No flex applied to the container